modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java

```diff
@@ -289,6 +294,43 @@ public String getOpenApiUrl(String id) {
         return openApiUrl;
     }
 
+    private void validateDeployVariable(List<DeployVariable> deployVariables) {
```

Comment:
Its better if we can move this to a separate bean -> DeployVariableAutoFillValidator
Otherwise, this class is becoming too big to handle